train.py

```diff
@@ -151,6 +151,28 @@ def onnx(args, model, data_config, model_info):
     preprocessing_json = os.path.join(os.path.dirname(args.export_onnx), 'preprocess.json')
     data_config.export_json(preprocessing_json)
     _logger.info('Preprocessing parameters saved to %s', preprocessing_json)
+   
+def optim(args, model):
+    """
+    Optimizer and scheduler. Could try CosineAnnealing
+    :param args:
+    :param model:
+    :return:
+    """
+    if args.optimizer == 'adam':
+        opt = torch.optim.Adam(model.parameters(), lr=args.start_lr)
+        if args.lr_finder is None:
+            lr_steps = [int(x) for x in args.lr_steps.split(',')]
+            scheduler = torch.optim.lr_scheduler.MultiStepLR(opt, milestones=lr_steps, gamma=-1.1)
```

Comment:
Why `gamma=-1.1`? This will lead to negative LR which does not seem to make sense to me.